### PR TITLE
Template changes 202405.1.0

### PR DESCRIPTION
**Pull request by srajeshwari-conga**
End-to-end testing has been done.

### CODE REVIEW

File: src/styles.scss [hunk 1]
```diff
@@ -1096,10 +1093,6 @@ div.fixed-top {
   background-color: white !important;
 }
 
-.dropdown-toggle::after {
-  display: none !important;
```
Comment (dboppana-conga) [inline src/styles.scss:1100]: https://github.com/congarevenuecloud/partner-commerce/pull/14.. This PR in partner commerce template you have this change.Can u=you reverify if we need this in both template or not?

Comment (srajeshwari-conga) [reply src/styles.scss:1100]: The same class was added thrice here so 2 of them were deleted.

**Review by mpatidar-conga — APPROVED** [review]

File: src/app/modules/assets/list/asset-list.component.ts [hunk 2]
```diff
@@ -322,7 +322,7 @@ export class AssetListComponent implements OnInit, OnDestroy {
       {
         icon: 'fa-dollar-sign',
         massAction: false,
-        label: 'ASSET_ACTIONS.BUY_MORE',
+        label: 'Buy More',
```
Comment (vbhan-conga) [inline src/app/modules/assets/list/asset-list.component.ts:325]: Why did we remove picking label from translation file ?

File: src/app/modules/assets/list/asset-list.component.ts [hunk 3]
```diff
@@ -338,7 +338,7 @@ export class AssetListComponent implements OnInit, OnDestroy {
       {
         icon: 'fa-ban',
         massAction: false,
-        label: 'ASSET_ACTIONS.TERMINATE',
+        label: 'Terminate',
```
Comment (vbhan-conga) [inline src/app/modules/assets/list/asset-list.component.ts:341]: Why did we remove picking label from translation file ?

Comment (srajeshwari-conga) [reply src/app/modules/assets/list/asset-list.component.ts]: changes added.

**Review by dboppana-conga — APPROVED** [review]

File: src/app/modules/assets/list/asset-list.component.ts (hunk 3 above)
Comment (srajeshwari-conga) [reply src/app/modules/assets/list/asset-list.component.ts]: Screenshot 2024-06-04 124421

**Review by nnaik-conga — APPROVED** [review]

**Review by vbhan-conga — APPROVED** [review]

**Pull request merged**